src/extension.ts

```diff
@@ -111,6 +112,7 @@ export async function activate(context: vscode.ExtensionContext) {
       absolute = path.join(
         context.extensionPath,
         "parsers",
+        "out",                    // bafflingly, upgrading the vscode-parse-tree dep to 0.4.0 causes the parsers to be placed in a different location???
```

Comment:
You do not need this. This occurs when you run [`cp`](https://github.com/rinOfTheStars/vscode-parse-tree/blob/af4b04747118f798af739be4081b597068db58e9/copy-assets.sh#L1) and the target directory already exists. Starting with a clean workspace and there is no out folder.

Reply:
I just created it; you didn't miss it. Thank you :)